Sources/Build/Command.swift

```diff
@@ -8,7 +8,13 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
+/// A command represents an atomic unit of build system work.
 struct Command {
-    let node: String
+    /// A unique name for the command.  This need not match any of the outputs
+    /// of the tool.
+    let name: String
```

Comment:
Might be worth noting that this defines the stable identifier which is used to match up incremental build records.